zebedee-reader/src/main/java/com/github/onsdigital/zebedee/search/indexing/Indexer.java

```diff
@@ -131,8 +132,10 @@ private void loadDepartments() throws IOException {
 
         elasticSearchLog("Indexing departments").log();
         long start = System.currentTimeMillis();
-        InputStream resourceStream = SearchBoostTermsResolver.class.getResourceAsStream("/search/departments/departments.txt");
-        try (BufferedReader br = new BufferedReader(new InputStreamReader(resourceStream))) {
+        try (
+                InputStream resourceStream = SearchBoostTermsResolver.class.getResourceAsStream(DEPARTMENTS_PATH);
+                BufferedReader br = new BufferedReader(new InputStreamReader(resourceStream))
+        ) {
```

Comment:
Consider declaring the InputStreamReader in the try with resources so it is closed if the BufferedReader constructor throws an exception.